components/ai_chat/resources/untrusted_conversation_frame/components/assistant_response/rich_search_widget.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { loadTimeData } from '$web-common/loadTimeData'
+import * as React from 'react'
+import styles from './rich_search_widget.module.scss'
+
+const RICH_SEARCH_WIDGETS_ORIGIN = loadTimeData.getString(
+  'richSearchWidgetsOrigin',
+)
+export default function RichSearchWidget(props: { jsonData: string }) {
+  if (!RICH_SEARCH_WIDGETS_ORIGIN) {
```

Comment:
I'm returning null here rather than in the parent component just to keep everything in once place.